<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sound</class>
 <widget class="QWidget" name="Sound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Net</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IPLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>10.10.10.41</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_11">
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Rx</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="3">
       <widget class="QProgressBar" name="RxEthBufUsageProgressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Tx</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="3">
       <widget class="QProgressBar" name="TxEthBufUsageProgressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>328</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="connectPushButton">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="listenPushButton">
        <property name="text">
         <string>Listen</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="micGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Audio mic</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2" colspan="2">
       <widget class="QSpinBox" name="sampleRateSpinBox">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Число каналов</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Частота дескритизации</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Имя файла</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="filenameLineEdit">
        <property name="text">
         <string>./snd_mic.raw</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="recordPushButton">
        <property name="text">
         <string>Запись</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="channelsNumberComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QPushButton" name="record5secPushButton">
        <property name="text">
         <string>Запись 5 сек</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="OnMicPushButton">
        <property name="text">
         <string>On</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QComboBox" name="resolutionComboBox">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Разрядность</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QProgressBar" name="micBufferUsageProgressBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
        <property name="format">
         <string>%v</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QComboBox" name="audioInputsComboBox">
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Устройство</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="telGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Audio tel</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Разрядность</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Имя файла</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QComboBox" name="resolutionComboBox_2">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="OnTelPushButton">
        <property name="text">
         <string>On</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="filenameLineEdit_2">
        <property name="text">
         <string>./snd_tel.raw</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Частота дескритизации</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Число каналов</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="channelsNumberComboBox_2">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QSpinBox" name="sampleRateSpinBox_2">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QProgressBar" name="telBufferUsageProgressBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
        <property name="format">
         <string>%v</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="recordPushButton_2">
        <property name="text">
         <string>Запись</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QPushButton" name="record5secPushButton_2">
        <property name="text">
         <string>Запись 5 сек</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QComboBox" name="audioOutputsComboBox">
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Устройство</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="logTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
